tools/deb/istio-start.sh

```diff
@@ -44,8 +40,20 @@ if [ -z "${POD_NAME:-}" ]; then
   POD_NAME=$(hostname -s)
 fi
 
+
+# Init option will only initialize iptables. Can be used
+if [[ ${1-} == "init" || ${1-} == "-p" ]] ; then
+  # Update iptables, based on current config. This is for backward compatibility with the init image mode.
+  # The sidecar image can replace the k8s init image, to avoid downloading 2 different images.
+  ${ISTIO_BIN_BASE}/istio-iptables.sh $*
```

Comment:
I'm adding tests - some of the commands are intended for reusing the script in k8s. 
We need to have some flexibility here - in future we can replace the error prone shell script 
with an error prone golang binary.

kube-proxy does all the iptables manipulation in go, and seems to work with CAP_NET_ADMIN only - but it's too complex and something like that is better in separate PR.